src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
@nLight please take a look at this comment. https://github.com/dcos/dcos-ui/pull/2261#discussion_r122276957

I have tried that solution and also other options with js but none seemed.

The solution in place using css is very specific/edge (take a look in the selector hierarchy) case and won't bleed out to another markup. I can always add more specificity using `> div::first-child`.